ODIN_II/SRC/read_blif.cpp

```diff
@@ -579,11 +581,105 @@ void create_hard_block_nodes(hard_block_models* models, FILE* file, Hashtable* o
     vtr::free(names);
 }
 
+/*---------------------------------------------------------------------------------------------
```

Comment:
@djns99 while this is rather ugly (don't know that I have a better way to write this tho)
this should do what you want now.
It still kind of breaks the spec, but this is the only way I can get the 'z' to go through during simulation.